test/extended/util/prometheus/helpers.go

```diff
@@ -143,29 +143,14 @@ func LocatePrometheusUsingRoutes(oc *exutil.CLI) (queryURL, prometheusURL, beare
 }
 
 func GetPrometheusSABearerToken(oc *exutil.CLI) string {
-	var bearerToken string
-	waitForServiceAccountInNamespace(oc.AdminKubeClient(), "openshift-monitoring", "prometheus-k8s", 2*time.Minute)
```

Comment:
Is this a load-bearing wait?

Reply:
I had assumed `waitForServiceAccountInNamespace` was polling until the ServiceAccount exists, but I see now that it is actually starting a watch and waiting for the ServiceAccount to have secrets populated, so it is totally redundant.